2/art/main.py

```diff
@@ -69,11 +69,11 @@
 
 # Solve LSE with ART
 start_time = timeit.default_timer()
-x_art = art(A, b, iterations)
+x_art = art(A, b, iterations,1)
```

Comment:
If relax is 1, then it would not make a difference wouldn't it?

Reply:
Didnt want to break already working code